CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # CHANGELOG
 
+## v0.11.0 - 2020-06-??
+
+Features:
+
+-  **(breaking-change)** Use `crystal:` property to filter candidates version. ([#395](https://github.com/crystal-lang/shards/pull/395), thanks @waj, @bcardiff)
```

Comment:
Probably should update https://github.com/bcardiff/shards/blob/changelog/0.11.0/SPEC.md#crystal to say it's not just informational now, and what the semantics of it are.

Reply:
I added them in this PR 🙈